main.py

```diff
@@ -38,7 +38,7 @@
 app.config['SESSION_PERMANENT'] = True
 app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(hours=2)
 app.secret_key = os.environ.get('FLASK_SECRET_KEY')
-PROJECT_ID = ""
+PROJECT_ID = "causal-impact-insights"
```

Comment:
should this be set on the environment level? maybe on the app.yaml and that would then allow the end user to configure the deployment without having to change the code